tensorflow/python/kernel_tests/nn_ops/conv_ops_test.py

```diff
@@ -2711,7 +2711,8 @@ def testInputGradientKernelSizeMatchesInputSize(self):
           padding="VALID",
           test_input=True,
           data_format=data_format,
-          use_gpu=use_gpu)
+          use_gpu=use_gpu,
+          max_err=0.005 if test.is_built_with_rocm() else 0.003)
```

Comment:
Why is the max error larger when you have more information for computing it?  What if use_gpu is `False` - then I'd expect the error to not change.  And have you tested with rocm to know there is a difference?

Reply:
Fixed